tox.ini

```diff
@@ -10,11 +10,11 @@ commands = test_plugin --standalone plugin.py
 commands = test_plugin --standalone plugin.py --regenerate
 
 [testenv:package]
-deps = hansken-extraction-plugin==0.8.3
+deps = hansken-extraction-plugin==0.9.1
```

Comment:
Only if it is a small task: it seems that `0.9.2` has been released as well. Perhaps increment this version in the PR?